go.mod

```diff
@@ -17,3 +17,5 @@ require (
 	google.golang.org/genproto v0.0.0-20200831141814-d751682dd103
 	google.golang.org/grpc v1.31.1
 )
+
+replace github.com/fxamacker/cbor/v2 => github.com/turbolent/cbor/v2 v2.2.1-0.20200911003300-cac23af49154
```

Comment:
Good catch! I'm pretty close to getting this upstreamed, so hopefully we can remove it again soon